pkg/server/build.go

```diff
@@ -0,0 +1,59 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"runtime"
+
+	"github.com/pyroscope-io/pyroscope/pkg/build"
+)
+
+type buildInfoJSON struct {
+	GOOS              string `json:"goos"`
+	GOARCH            string `json:"goarch"`
+	Version           string `json:"version"`
+	ID                string `json:"id"`
+	Time              string `json:"time"`
+	GitSHA            string `json:"gitSHA"`
+	GitDirty          int    `json:"gitDirty"`
+	UseEmbeddedAssets bool   `json:"useEmbeddedAssets"`
+}
+
+func (ctrl *Controller) buildHandler(w http.ResponseWriter, r *http.Request) {
+	buildInfoStr, err := buildInfoJSONString(true)
+	if err != nil {
+		renderServerError(w, fmt.Sprintf("could not marshal buildInfoObj json: %q", err))
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(200)
```

Comment:
Here and below: `http.StatusOK` is set by default, this line may be safely omitted.